docs/servers/add-digitalocean-server.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Add a Server to DigitalOcean
+intro: Devopness allows you to provision a server on DigitalOcean using Droplets and manage it through Devopness.
+links:
+    overview:
+    quickstart:
+    previous:
+        - /docs/servers
+    next:
+        - /docs/servers/check-server-provision-logs
+    guides:
+    related:
+    featured:
+---
+
+> If you don't have an DigitalOcean credential, please follow the guide [Add DigitalOcean Credential](/docs/cloud-provider-credentials/add-digitalocean-credential/)
+
+1. On Devopness, navigate to a project then select an environment
+1. If there's no other server in the selected environment, just click `ADD SERVER`
+1. If a list of resource cards is displayed, find the `Servers` card and click View on it
+1. On the upper-right corner of the list click `ADD SERVER`
+1. Choose `DigitalOcean` as cloud provider
+1. Select an DigitalOcean credential using the drop-down menu
+    > If no credential is listed or you want to use a different one, please follow the [Add a Cloud Provider Credential](/docs/cloud-provider-credentials/add-cloud-provider-credential/) guide
+1. Click `NEXT`
+1. Select an avaliable region using the drop-down menu
+1. Select an avaliable instance type using the drop-down menu
```

Comment:
It seems `available` has been misspelled as `avaliable` here and also in other docs. If you could fix them on this PR as well, that would be great. See list here: https://github.com/search?q=repo%3Adevopness%2Fdevopness%20avaliable&type=code

```suggestion
1. Select an available region using the drop-down menu
1. Select an available instance type using the drop-down menu
```

Reply:
I'll update the PR soon with the changes applied